### PR TITLE
[15.0][MIG] stock_picking_import_serial_number: Migration to version 15.0

### PR DESCRIPTION
**Pull request by carolinafernandez-tecnativa**
- Standard migration to V15

@Tecnativa

TT45386

@pedrobaeza @sergio-teruel

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: stock_picking_import_serial_number/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 Tecnativa - Sergio Teruel
+# Copyright 2023 Tecnativa - Carolina Fernandez
```
Comment (sergio-teruel) [inline stock_picking_import_serial_number/__manifest__.py:2]: Please remove from files...

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

File: stock_picking_import_serial_number/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply stock_picking_import_serial_number/__manifest__.py:2]: This is correct to be put.

**Review by Saran440 — APPROVED** [review]
LGTM :+1:

**Comment by carolinafernandez-tecnativa** [comment]
@sergio-teruel @pedrobaeza Is it okay to move on ?

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**